members/CAL000481.yaml

```diff
@@ -48,7 +46,6 @@ contact_form:
         - value: Submit
           selector: "#submitButton"
   success:
-    headers:
-      status: 200
     body:
       excludes: "Select an Issue"
+      contains: "Thank you for contacting the office"
```

Comment:
is this one supposed to have a lower-case 'o' ?

Reply:
yup, https://gyazo.com/07624144af0b9e59862f2b4f1d48519d